src/main/java/mate/academy/springboot/datajpa/service/mapper/CategoryMapper.java

```diff
@@ -0,0 +1,22 @@
+package mate.academy.springboot.datajpa.service.mapper;
+
+import mate.academy.springboot.datajpa.dto.request.CategoryRequestDto;
+import mate.academy.springboot.datajpa.dto.response.CategoryResponseDto;
+import mate.academy.springboot.datajpa.model.Category;
+import org.springframework.stereotype.Component;
+
+@Component
+public class CategoryMapper {
```

Comment:
Don't forget about abstraction. Let's create an interface for RequestDtoMapper and ResponseDtoMapper